core/tests/protractor_utils/ExplorationEditorMainTab.js

```diff
@@ -156,10 +156,8 @@ var ExplorationEditorMainTab = function() {
     var isVisible = await editorWelcomeModal.isPresent();
 
     if (isVisible) {
-      await waitFor.elementToBeClickable(
-        dismissWelcomeModalButton,
-        'Welcome modal is taking too long to appear');
-      await dismissWelcomeModalButton.click();
+      await action.click(
+        'Dismiss Welcome Modal not showing up', dismissWelcomeModalButton);
```

Comment:
The first arg to `action.click` should just be the name, not the "not showing up" part

Reply:
Done!